src/server/pps/server/pipeline_controller.go

```diff
@@ -69,7 +75,7 @@ type pipelineOp struct {
 	crashingMonitorCancel func()
 
 	bumpChan  chan struct{}
-	opManager *opManager
+	opManager *pcManager
```

Comment:
would it make sense to rename opManager -> pcMgr as well? (per the change in the ppsMaster?)